cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/store/AppMetadataStore.java

```diff
@@ -2567,11 +2690,6 @@ private void writeApplicationSerialized(String namespaceId, String appId, String
           change.getDescription()));
     }
     fields.add(Fields.booleanField(StoreDefinition.AppMetadataStore.LATEST_FIELD, markAsLatest));
-
-    if (sourceControlMeta != null) {
```

Comment:
if sourceControlMeta shouldn't be stored in the table anymore, it should be removed from the method arguments as well.

Reply:
Got it. Done